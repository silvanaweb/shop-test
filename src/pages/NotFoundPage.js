import React from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
  <div className="content-container">
    Not found - <Link to="/">Go home</Link>
  </div>
);

export { NotFoundPage };
