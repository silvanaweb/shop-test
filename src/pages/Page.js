import React from "react";
import { Route } from "react-router-dom";
import Header from "../components/Header";

const Page = ({ component: Component, ...rest }) => (
  <Route
    {...rest}
    component={props => (
      <div>
        <Header {...props} />
        <div className="content-container">
          <Component {...props} />
        </div>
      </div>
    )}
  />
);

export { Page };
