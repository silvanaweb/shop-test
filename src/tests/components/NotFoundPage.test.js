import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";
import { NotFoundPage } from "../../pages/NotFoundPage";

test("should render NotFoundPage correctly", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
