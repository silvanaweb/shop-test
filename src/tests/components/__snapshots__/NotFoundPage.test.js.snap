// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage correctly 1`] = `
<div
  className="content-container"
>
  Not found - 
  <Link
    to="/"
  >
    Go home
  </Link>
</div>
`;
